develop/items/custom-armor.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Custom Armor
+description: Learn how to create your own armor sets.
+authors:
+  - IMB11
+---
+
+# Custom Armor
+
+Armor provides the player with increased defense against attacks from mobs and other players.
+
+## Creating an Armor Material
```

Comment:
Maybe mention the vanilla `ArmorMaterials` class somewhere here, to give devs context in terms of values?